figures/make_ngc6977_image.py

```diff
@@ -1,21 +1,75 @@
 from astropy.io import fits
+import astropy.units as u
 from astropy.utils.data import get_pkg_data_filename
-from astropy.visualization import make_lupton_rgb
+from astropy.visualization import make_lupton_rgb, astropy_mpl_style
+from astropy.wcs import WCS
+import matplotlib.pyplot as plt
+plt.style.use(astropy_mpl_style)
+plt.rcParams['font.family'] = plt.rcParamsDefault['font.family']
+plt.rcParams['lines.marker'] = ''
+from reproject import reproject_interp
+import numpy as np
 
 # load the data
-g_name = get_pkg_data_filename('visualization/reprojected_sdss_g.fits.bz2')
-r_name = get_pkg_data_filename('visualization/reprojected_sdss_r.fits.bz2')
-i_name = get_pkg_data_filename('visualization/reprojected_sdss_i.fits.bz2')
-g = fits.open(g_name)[0].data
-r = fits.open(r_name)[0].data
-i = fits.open(i_name)[0].data
+# g_name = get_pkg_data_filename('visualization/reprojected_sdss_g.fits.bz2')
+# r_name = get_pkg_data_filename('visualization/reprojected_sdss_r.fits.bz2')
+# i_name = get_pkg_data_filename('visualization/reprojected_sdss_i.fits.bz2')
+g_name = get_pkg_data_filename('hash/fb425fccf6326e34f4566711d8910364')
+r_name = get_pkg_data_filename('hash/e6670142bc3c83936d84bd5d5d331d6a')
+i_name = get_pkg_data_filename('hash/ed6e4146ea0a1d2af0f005dd8ba79282')
+
```

Comment:
Using the hash here is not reproducable, except by you I think. The same might be true for the other file names, so I don't think that's a problem, but worth mentioning.

Reply:
Oops, the hashes were leftover because of some issues I was having with the data server - the other (commented out) filenames are reproducible because they download from data.astropy.org. Will switch back